google/cloud/bigtable/table.py

```diff
@@ -506,7 +506,9 @@ def read_row(self, row_key, filter_=None):
         """
         row_set = RowSet()
         row_set.add_row_key(row_key)
-        result_iter = iter(self.read_rows(filter_=filter_, row_set=row_set))
+        result_iter = iter(
+            self.read_rows(filter_=filter_, row_set=row_set, overall_timeout=60)
```

Comment:
is it ok to set a default overall_timeout?

Reply:
Is this the dafult from DEFAULT_RETRY_READ_ROWS.deadline? If so, can we use that instead?

Reply:
No, this is the default unary deadline from the service json file

Reply:
Can the `60` be retrieved via configuration or in env var instead of default like this?

Reply:
i bubbled it up as a kwarg to the method